### DISCUSSION TITLE
how can I disble new tab on middle click in tab list (scroll bar of tree tabs) ?

### DISCUSSION THREAD

**Discussion by SequentialDesign** [piroor/treestyletab#3433] [General]
I want to use my middle click on the tab list scroll bar, but a new tab is opened each time I do it.  

how can I get rid of this problem ?

**Comment by piroor** [comment]
Currently there is no special option for middle click on the scrollbar. TST is designed to treat middle click on the scrollbar same as the one on tabbar's blank area and the new tab button. You just can control how newly opened tab is placed but you cannot control opening a new tab or not. This behavior is hardcoded.

Developing a helper addon with TST's API may become a solution for you. TST has ability to notify events to addons when the blank area is clicked, and you can cancel opening a new tab based on the coordinates of the click event.
https://github.com/piroor/treestyletab/wiki/API-for-other-addons#when-the-non-tab-area-is-clicked
